packages/landingpage-new/pages/home/component-sample/component-sample.tsx

```diff
@@ -0,0 +1,201 @@
+import styles from './component-sample.module.scss';
+import {
+  InoChip,
+  InoRange,
+  InoSnackbar,
+  InoInput,
+  InoSwitch,
+  InoControlItem,
+  InoSegmentGroup,
+  InoCard,
+  InoButton,
+  InoIcon,
+  InoList,
+  InoSegmentButton,
+} from '@elements';
+import { useState } from 'react';
+
+export default function ComponentSample() {
+  const [inputValue, setInputValue] = useState('');
+  const [rangeValue, setRangeValue] = useState(50);
+  const [checkboxValue, setCheckboxValue] = useState(false);
+  const [radioboxValue, setRadioboxValue] = useState(false);
+  const [switchValue, setSwitchValue] = useState(false);
+  const [segmentButtonValue, setSegmentButtonValue] = useState('opt-2');
+  const [showSnackbar, setSnackbar] = useState(false);
+  const [showChip, setChip] = useState(true);
+
+  return (
+    <div>
+      <div className={styles.header}>
+        <p>
+          the <b>components</b>
+        </p>
+      </div>
+      <div className={styles.subheader}>
+        <p className={styles.subheader}>
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr Lorem ipsum
+          dolor sit amet, consetetur sadipscing elitr
+        </p>
+      </div>
+      <div className={styles.container}>
+        <div className={styles.chip}>
+          <InoCard disableElevation={true}>
+            <div className={styles.content} slot="content">
+              {showChip && (
+                <InoChip
+                  removable={true}
+                  onChipRemoved={() => {
+                    setChip(false);
+
+                    setTimeout(() => {
+                      setChip(true);
+                    }, 5000);
+                  }}
+                >
+                  Removable Chip
+                </InoChip>
+              )}
```

Comment:
IMO the removable variant is not suited for showing off the chips functionality because it can be irritating if the chip suddenly disappears. Maybe we should [use the filter functionality](https://elements.inovex.de/version/v7.1.2/?path=/docs/buttons-ino-chip--playground#filter) instead? Two chips with a short label (e.g. Pizza, Burger) should be enough

![image](https://user-images.githubusercontent.com/22963121/202193648-e7df5451-bfef-47d5-bd9a-350408943f15.png)